### DISCUSSION TITLE
compatible hardware?

### DISCUSSION THREAD

**Discussion by jandrefrey** [Zer0-bit/gaggiuino#132] [Q&A] [answered]
good day.
i would like to thank you for this inspiring mod and all the effort you have out into creating this.
i have a question, however: 
can s normal arduino nano be used here and are there any specific changes in the setup for it to work?
here is a link to the one I am looking at:[] (https://www.robotics.org.za/NANO-ORG)

also, will this current sensor work in the situation:[] (https://www.robotics.org.za/4031)

**Comment by Zer0-bit** [comment, marked as answer]
Hi, it's a yes to both questions.

**Reply by jandrefrey** [reply]
awesome, thanks so much!!

**Comment by Deltad** [comment]
I apologize for the stupid question.
In the shopping list, the power supply module "12v to 5V step-down" is no longer available for purchase.
What can it be replaced by?

**Reply by itsmepetrov** [reply]
@Zer0-bit could you please give a new link for step-down?

**Comment by Zer0-bit** [comment]
1. Pls don't use discussions marked as answered for new questions
2. Changed the step-down in the BOM but it's not smth I or anyone tested so if it's not going to provide the necessary voltage stability you  will have to change it again.
